routes/users.js

```diff
@@ -2,30 +2,22 @@ var express = require("express");
 var router = express.Router();
 var db = require("../db.js");
 
-router.route("/")
+router
+  .route("/")
   .get(function (req, res) {
-    res.statusCode = 200;
-    res.json({
-      "id": 1,
-      "display_name": "BobJoe",
-      "name": "Robert Joseph"
-    })
-  })
-  .put(function(req, res) {
-    res.statusCode = 200;
-    res.end();
+    // Get all users
```

Comment:
We probably shouldn't have a route where we return all users. I think we should delete this route and update the README.md